### DISCUSSION TITLE
ERROR #98124  WEBPACK  Generating development SSR bundle failed  Can't resolve '@reach/router/lib/utils' in '/Users/marcoadinolfi/Documents/dalyriumbay-sito/.cache'

### DISCUSSION THREAD

**Discussion by MarcoAdinolfi** [gatsbyjs/gatsby#33852] [Help]
Hello,
just after upgrading gatsby-cli from 3.14.0 to 3.14.1 and npm to 8.1.2, gatsby develop keep failing, it gives this error:
`ERROR #98124  WEBPACK

Generating development SSR bundle failed

Can't resolve '@reach/router/lib/utils' in
'/Users/marcoadinolfi/Documents/dalyriumbay-sito/.cache'

If you're trying to use a package make sure that '@reach/router/lib/utils' is
installed. If you're trying to use a local file make sure that the path is
correct.

File: .cache/find-path.js`

I tried to restore the system as it was before by copying all the old dependecies in package.json and package-lock.json but it still doesn't work.
I also already tried with gatsby clean.
Does anyone know why does it happen? What can I do make it work again?

Further info:
Gatsby CLI version: 4.1.0
Gatsby version: 2.32.13
NPM version: 8.1.2
Node version: 14.16.0

Thank you

**Comment by LekoArts** [comment]
Try this:

- `gatsby clean`
- Delete `node_modules` and `package-lock.json`
- Run `npm install --legacy-peer-deps`

**Reply by MarcoAdinolfi** [reply]
Hello,
Thank you for your reply.
I've already tried it but it didn't work, at the end I managed to fix it by restoring an old directory version that I have, but I still don't know what created the problem so I'm not completely satisfied.
Thank you anyway for your help

**Reply by beingraigari** [reply]
Thanks @LekoArts  for your suggestions. 
`npm install --force `worked for me. Though this is not the right way to avoid the error it can save you for once.
